go/oasis-test-runner/oasis/oasis.go

```diff
@@ -811,7 +812,8 @@ func nodeLogPath(dir *env.Dir) string {
 }
 
 func internalSocketPath(dir *env.Dir) string {
```

Comment:
I don't think we should just make `internalSocketPath` return an abstract address. Consider renaming all `*SocketPath` to `*SocketAddress`.

Reply:
Should I also start prepending `unix:` prefix everywhere?